app/assets/styles/partials/components/_table.scss

```diff
@@ -0,0 +1,32 @@
+.table {
+
+}
+caption {
+  text-align: left;
+}
+.table__dense {
+  font-size: 81.25%;
+
+.table--header--cell {
+    padding: 0.5rem 0 0.25rem 1rem;
+    @include nth-element(1, 0);
+  }
+}
+
+.table--row {
+  border-bottom: 1px solid $color-grey;
+}
+
+.table--header--cell {
+  padding: 0.5rem 0 0.5rem 1rem;
+  border-bottom: 2px solid $color-very-dark-grey;
+  @include nth-element(1, 0);
+}
+.table--cell__numeric {
+  text-align: right;
+}
+
+.table--cell {
```

Comment:
The designs have a smaller font size for the table cells. You can see it applied in the prototype where it sets it to 12pt. That's probably not how we want to implement it in runners CSS but we certainly want the same behaviour.